client/my-sites/post-share/publicize-actions-list.jsx

```diff
@@ -22,18 +22,20 @@ import {
 	SCHEDULED,
 	PUBLISHED,
 } from './constants';
+import SectionNav from 'components/section-nav';
+import NavTabs from 'components/section-nav/tabs';
+import NavItem from 'components/section-nav/item';
 
 class PublicizeActionsList extends PureComponent {
 	static propTypes = {
-		section: PropTypes.string,
 		siteId: PropTypes.number,
 		postId: PropTypes.number,
```

Comment:
While we're in here how about adding `publishedActions` and `scheduledActions` to `propTypes`